--- /opt/etag/config/api_config.py
+++ /opt/etag/config/api_config.py
@@ -49,12 +49,27 @@
 # https://docs.djangoproject.com/en/1.6/ref/settings/#databases
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        #'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        #'NAME': 'etag_auth',
+        #'USER': 'etag_master',
+        #'PASSWORD': 'etag_master',
+        #'HOST': 'cybercom_postgres',
+        #'PORT': '5432',
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    },
+    'etag': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'etag',
+        'USER': 'etag_master',
+        'PASSWORD': 'etag_master',
+        'HOST': 'cybercom_postgres',
+        'PORT': '5432',
     }
 }
 
-DATABASE_ROUTERS = []
+#DATABASE_ROUTERS = []
+DATABASE_ROUTERS = ['etag.etag_router.EtagRouter',]
 #******* Queue  *******************************************************
 
 MEMCACHE_HOST = "cybercom_memcache"
